spec/requests/api/tags_spec.rb

```diff
@@ -60,17 +60,17 @@
     post 'Creates a tag group' do
       tags 'Tags'
       consumes 'application/json'
-      expected_request_schema = SpecSchemas::TagGroupCreateRequest.new
+      expected_request_schema = SpecSchemas::SpecLoader.new('tag_group_create_request').load
```

Comment:
Another opportunity for a helper method -- just sayin' ;). Something like:

```ruby
expected_request_schema = load_spec_schema('tag_group_create_request')
```

Reply:
Added a helper in https://github.com/discourse/discourse/pull/11849